templates/gallery/showImage.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+var_dump($image);
+die;
```

Comment:
debug code should not be commited in the repository

Reply:
oups